From 58c99a5637e390466750ddbe33e7a58436e71032 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 13 Jan 2020 15:21:33 -0600
Subject: [PATCH 4/4] arm64: dts: logicpd-imx8mm-kit: Point flexspi to proper
 node

The U-Boot and Kernel have slighly different DTSI files for
the i.MX8M Mini SOC.  The flexspi was removed as the node didn't
match what U-Boot expects.

This patch uncomments-out the flexspi node, and names it using
the node name specified in the fsl-imx8mm.dtsi file

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm/dts/lpd-imx8mm-som.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/lpd-imx8mm-som.dtsi b/arch/arm/dts/lpd-imx8mm-som.dtsi
index 643f6a4..74bda0c 100644
--- a/arch/arm/dts/lpd-imx8mm-som.dtsi
+++ b/arch/arm/dts/lpd-imx8mm-som.dtsi
@@ -25,7 +25,7 @@
 	};
 };
 
-/* &flexspi {
+&flexspi0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_flexspi0>;
 	status = "okay";
@@ -38,7 +38,7 @@
 		spi-max-frequency = <29000000>;
 		spi-nor,ddr-quad-read-dummy = <6>;
 	};
-}; */
+};
 
 &i2c1 {
 	clock-frequency = <400000>;
-- 
2.17.1

